engine/apps/alerts/paging.py

```diff
@@ -0,0 +1,165 @@
+from typing import Any
+
+from django.db import transaction
+from django.db.models import Q
+
+from apps.alerts.models import Alert, AlertGroup, AlertGroupLogRecord, AlertReceiveChannel, UserHasNotification
+from apps.alerts.tasks.notify_user import notify_user_task
+from apps.schedules.ical_utils import list_users_to_notify_from_ical
+from apps.schedules.models import OnCallSchedule
+from apps.user_management.models import Organization, Team, User
+
+USER_HAS_NO_NOTIFICATION_POLICY = "USER_HAS_NO_NOTIFICATION_POLICY"
+USER_IS_NOT_ON_CALL = "USER_IS_NOT_ON_CALL"
+
+# notifications: (User|Schedule, important)
+UserNotifications = list[tuple[User, bool]]
+ScheduleNotifications = list[tuple[OnCallSchedule, bool]]
+
+
+def _trigger_alert(organization: Organization, team: Team, title: str, message: str, from_user: User) -> AlertGroup:
+    """Trigger manual integration alert from params."""
+    alert_receive_channel = AlertReceiveChannel.get_or_create_manual_integration(
+        organization=organization,
+        team=team,
+        integration=AlertReceiveChannel.INTEGRATION_MANUAL,
+        deleted_at=None,
+        defaults={
+            "author": from_user,
+            "verbal_name": f"Manual incidents ({team.name if team else 'General'} team)",
```

Comment:
Probably change "incidents" to "alert groups"?

Reply:
Yeah, sounds good, changing.